src/components/admin/DataTable.vue

```diff
@@ -75,22 +112,74 @@
 <script lang="ts" setup>
 import { computed, onMounted, onUnmounted, ref } from 'vue';
 import { useRequestStore } from 'src/stores/request.store';
-import { IRequest, IProfile } from 'src/entities';
+import { IRequest } from 'src/entities';
 import { date, QTableColumn } from 'quasar';
 
 import { TheDialogs } from 'src/dialogs/the-dialogs';
+import { RouteLocationRaw, useRoute, useRouter } from 'vue-router';
+import { update } from 'firebase/database';
 
-const props = defineProps<{
-  type: IProfile['type'];
-}>();
-
+const $router = useRouter();
+const $route = useRoute();
+// const props = defineProps<{ type: IProfile['type'] }>();
 const requestStore = useRequestStore();
-
 const applications = computed(() => {
   return requestStore.requests;
 });
 
-const filter = ref<IRequest['status']>('pending');
+interface filterOption {
+  label: string;
+  value: string;
+  to?: string;
+  icon?: string;
+  color?: string;
+}
+
+const routePath = $route.path.split('/')[2];
```

Comment:
reading this is vague, better to use `$route.params.type` which will require to change your route configuration. 
 ```suggestion
const routeApplicationType = $route.param.type as string;
```

Reply:
you need to update your route as follows without sub route to simplify it:
```ts
     {
        path: 'application/:type',
        name: 'application',
        component: () => import('pages/admin/ApplicationPage.vue'),
        meta: {
          requiresLogin: ['admin'],
          menu: 'Application',
          title: 'Application Page',
          icon: 'work_outline',
       },
```
removing the child routes